db/migrate/20160215223349_create_picks.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePicks < ActiveRecord::Migration
+  def change
+    create_table :picks do |t|
+      t.belongs_to :user, null: false
```

Comment:
I like your method. I think it's clear enough. I'll start using this in my migrations in the future